### ISSUE TITLE
Franz Dead?

### ISSUE THREAD

**Issue by Ansem93** [meetfranz/franz#1067]
Hi, the latest commist is from nearl 5 months ago.
Is Franz dead? :(

**Comment by adrienne** [comment]
Damn, the webpage is down, too. :( Guess i gotta find a different app.

**Comment by Ansem93** [comment]
The webpage is working fine for me, only no update for 5 months

**Comment by urbels** [comment]
There is only one alternative afaik and its also not well maintained..

**Comment by adrienne** [comment]
Someone has forked Rambox and is committing to actively maintaining and improving it: https://github.com/TheGoddessInari/rambox

**Comment by solid-pixel** [comment]
Yeah, pretty sure they've abandoned the ship. Would have been nice of them to speak.

Thanks @adrienne gonna give Rambox a try, although not a big fan of the UI, I really loved Franz's. But let's see.

What a shame!

**Comment by elkuku** [comment]
Too bad :disappointed:

**Comment by gaellelo** [comment]
I hope not, just Stefan the main developer posted a message on slack that he got sick on May 2nd and hasn't been active since then. It seems that he has not fully recovered yet... :(
We should send him tons of flowers :)

> Hey guys, quick update. I've caught a nasty virus infection a few weeks ago and unfortunately, recovery is painfully slow. I just wanted to let you know that we're not ignoring you but I need to get back on my feet before  operations will be back to normal.
https://meetfranz.slack.com/archives/C5ULLT74P/p1525284026000732

**Comment by solid-pixel** [comment]
If that's the case then I wish him all the best! A 4-month long virus sounds really serious, hope he's ok.

**Comment by Gaboris** [comment]
Áh, okay I just came by to ask exactly this, but it's good to see that there's some feedback. :)

IF Stefan is really in poor health and needs more time to recover I wish him the best and I can not wait to see what's next for this great app that I was looking for for so many years.

So far I'm fine with the current build so I'm not complaining but IF for whatever reason the project fails to start up again I'll look up that Rambox thing that was mentioned just in case a random update to any of the other systems might break Franz for me.

Again, I wish the best to all involved and I can NOT wait to hear about this project again. :slightly_smiling_face:

**Comment by Gaboris** [comment]
> 
> 
> > Someone has forked Rambox and is committing to actively maintaining and improving it: https://github.com/TheGoddessInari/rambox
> 
> I tried it, really bothers me that you need to pay for the dark theme. For that reason I'm gonna stick around here and hope they get active again.

Yeah, I checked them out as well but I'd need some help with this cuz I don' get what is the actual difference between this fork(OS) and the original(CE)? At least I didn't find any comprehensive list for it.

Looking at the actual Rambox, even though it's annoying that stuff like the dark theme is hidden behind a paywall it's completely understandable so my main complaint is the fact that it seems they collect user data.
Probably just analytics, but they don't say anything about it in any of their docs which is REALLY shady so staying far away from them until someone can confirm that they either stop it or at least make it clear what they are doing.

So I'm sticking around, but this did bring up a big question for me so I'll post a new "Issue" about it in a bit.

**Comment by urbels** [comment]
@Gaboris This repo is a copy from real rambox. You dont need to pay anything for it.

Rambox rewrote whole thing and now asking money for it. They left CE version as seperate and this guy forked it and made some improvements and fixes.

**Comment by Gaboris** [comment]
> 
> 
> @Gaboris This repo is a copy from real rambox. You dont need to pay anything for it.
> 
> Rambox rewrote whole thing and now asking money for it. They left CE version as seperate and this guy forked it and made some improvements and fixes.

Yes I got that much, but I guess you mean that that's the version frome BEFORE they split Rambox up into the payed Pro and free CE versions or that this is a copy of just CE with stuff changed?

My main problem is still about not having any form of list about what's really different, but still thanks.

**Comment by urbels** [comment]
Its just a copy from actual CE. But he made many changes already.

**Comment by Gaboris** [comment]
> 
> 
> Its just a copy from actual CE. But he made many changes already.

Again, I got that part, but okay thanks.
I'm actually trying to ask them about it over there to not derail this "thread" any further.

**Comment by gabamnml** [comment]
Franz its dead forever?

**Comment by sNiXx** [comment]
Any news on Franz's future?

**Comment by Gaboris** [comment]
OH! It seems like @adlk has made several merges from pull requests today.
Could this mean that we'll get to see a new update before the next year? Now I'm intrigued.

**Comment by Stanzilla** [comment]
Yeah looks like https://github.com/meetfranz/franz/commits/develop is getting updates. Let's hope he is back for good and the awesome changes @MikeDabrowski did can be pulled in

**Comment by Gaboris** [comment]
Okay these mass "stale"-ing by bots might be a bit concerning, but I'm hopeful these are more about cleaning up the issues list and focus on stuff that are of interest to more people or something. We'll see.

**Comment by closingin** [comment]
Hello, now that the project is active again, I think you can close this issue @Ansem93 ?

Also, this is not the right place to discuss about rambox, if you want to stay updated about Franz, you can join the slack [here](http://slack.franz.im/) :slightly_smiling_face:

**Comment by lolmaus** [comment]
> Hello, now that the project is active again

Is it? There have been some merges, but still no response from project owners. 😒

**Comment by madsmtm** [comment]
Hey everyone, there's no need for the grumpy comments, I'm sure the maintainers / project owners are getting around to answering you all! Remember, these folks are real people, with a life outside GitHub to deal with :wink:

**Comment by lolmaus** [comment]
@madsmtm, Franz is a commercial product. I've been subscribed for the paid account for three months and haven't received a single release, update, tweet or reply from owners, not mentioning that none of promised paid features haven't been worked on.

How hard it is to tweet something like "I'm dealing with some tough :hankey: IRL, will get back to you in a couple months"?

**Comment by lolmaus** [comment]
> A message has been sent on Slack to explain exactly this.

A message by whom? By you?

Still no official statement from @adlk or another Franz employee?

**Comment by blaise-io** [comment]
> A message has been sent on Slack to explain exactly this. 

Can you copy/paste that message here please?

**Comment by cpt-kernel** [comment]
I just got on the Franz train a few months ago, I love the project and will contribute $$ as soon as I see some activity. I'm hoping for the best. And would be nice if @closingin could post the message, as many of us likely don't want to join another slack channel.

**Comment by closingin** [comment]
Gaëlle already posted it above, @lolmaus (and others), it's a message from Stefan himself (adlk), but it seems like you didn't read. Also I don't think there are such things like Franz employees, but there are developers and contributors.

Anyway, people (should) know that with community-opened software, there's a chance for development to get slowed down from time to time. Just accept that its developers can take a break from it.. This debate won't get us anywhere, especially because there have been some activity on the project the past couple of days.

So keep calm and we'll see how it goes, I don't have more information than you all. :slightly_smiling_face: 

---
Quote:

> I hope not, just Stefan the main developer posted a message on slack that he got sick on May 2nd and hasn't been active since then. It seems that he has not fully recovered yet... :(
> We should send him tons of flowers :)
> 
> > Hey guys, quick update. I've caught a nasty virus infection a few weeks ago and unfortunately, recovery is painfully slow. I just wanted to let you know that we're not ignoring you but I need to get back on my feet before  operations will be back to normal.
> > https://meetfranz.slack.com/archives/C5ULLT74P/p1525284026000732

**Comment by lolmaus** [comment]
@closingin That message is from May 2 -- literally half a year ago.

As @Gaboris has noticed, during past few weeks, @adlk has been triaging PRs (see the list of PRs [sorted by recently updated](https://github.com/meetfranz/franz/pulls?q=is%3Apr+is%3Aclosed+sort%3Aupdated-desc)).

Looks Stefan feels well enough to review PRs, which requires a lot of mental effort and concentration. So he's neither dead, nor lost interest in Franz. We all hope Stefan has recovered from the illness.

It's still unclear why there has been no statement from him: no blog post, no tweet ([@franzmessenger](https://twitter.com/franzmessenger?lang=en), [@smalzner](https://twitter.com/smalzner?lang=en)), no newsletter email, not even a message on Slack.

**Comment by thejakobn** [comment]
I came across this issue (or thread by now) when searching for the reason why there are no more updates to Franz Messenger, as I heavily use it every day. I wish Stefan all the best in his recovery and hope that the development of this awesome project will continue somehow in the future :)

**Comment by lolmaus** [comment]
Gentlemen, there has been a beta release of Franz: v5.0.0-beta.19. See [the changelog](https://meetfranz.com/changelog/) for details.

Please **do not** discus the release here (features, bugs). This thread is for nagging about lack of communication from Franz team.
